### PR TITLE
Updated test suite, added more basic unit tests

### LINKED ISSUE #68: Write Unit Testing - Switch from PHPUnit 3 to PHPUnit 7/8
There is a single incomplete unit test in this library and it's using PHPUnit 3 which is incredibly deprecated. We should upgrade to PHPUnit 7/8 which is compatible with PHP 7.x and write unit tests for more than just the Address object.

### PR DESCRIPTION
**Pull request by Justintime50**
Updated PHPUnit from v3.x to v8.x, tweaked single previously included unit test and added a Parcel and shipment unit test. More to come but this is ready to merge if approved.

Closes #68

### CODE REVIEW

File: test/EasyPost/Test/ParcelTest.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace EasyPost\Test;
+
+use EasyPost\Parcel;
+use EasyPost\EasyPost;
+
+EasyPost::setApiKey('cueqNZUb3ldeWTNX7MU3Mel8UXtaAMUi');
```
Comment (jontsai) [inline test/EasyPost/Test/ParcelTest.php]: is this a testing API key? it probably shouldn't be checked into code, and be passed as an environment variable.

Something like this: https://docs.travis-ci.com/user/environment-variables/
And I think this: https://www.php.net/manual/en/function.getenv.php

File: test/EasyPost/Test/ShipmentTest.php [hunk 2]
```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace EasyPost\Test;
+
+use EasyPost\Shipment;
+use EasyPost\EasyPost;
+
+EasyPost::setApiKey('cueqNZUb3ldeWTNX7MU3Mel8UXtaAMUi');
```
Comment (jontsai) [inline test/EasyPost/Test/ShipmentTest.php]: don't check in API keys to code

File: test/EasyPost/Test/AddressTest.php [hunk 3]
```diff
@@ -7,67 +7,47 @@
 
 EasyPost::setApiKey('cueqNZUb3ldeWTNX7MU3Mel8UXtaAMUi');
```
Comment (jontsai) [inline test/EasyPost/Test/AddressTest.php]: load the API key using an environment variable

File: test/EasyPost/Test/ParcelTest.php (hunk 1 above)
Comment (att14) [reply test/EasyPost/Test/ParcelTest.php]: This is how all of the client libraries do it as of today. I think this is outside the scope of this change.

Comment (Justintime50) [reply test/EasyPost/Test/ParcelTest.php]: @jontsai I agree with @att14 as all libraries are setup this way currently, but I also agree with you that moving forward we should set these up as environment variables instead!

Comment (jontsai) [reply test/EasyPost/Test/ParcelTest.php]: sgtm

**Pull request merged**